gems/aws-sdk-autoscaling/lib/aws-sdk-autoscaling/activity.rb

```diff
@@ -21,6 +21,7 @@ def initialize(*args)
       @id = extract_id(args, options)
       @data = options.delete(:data)
       @client = options.delete(:client) || Client.new(options)
+      @waiter_block_warned = false
```

Comment:
These service files are generated. I see that you added this to the mustache template as well. Were these added manually or the result of `rake build`?

Reply:
`rake build`